[2024-02-25T10:17:02.196+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data_from_csv.csv_to_stg scheduled__2024-02-25T10:16:00+00:00 [queued]>
[2024-02-25T10:17:02.208+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data_from_csv.csv_to_stg scheduled__2024-02-25T10:16:00+00:00 [queued]>
[2024-02-25T10:17:02.209+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-02-25T10:17:02.220+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): csv_to_stg> on 2024-02-25 10:16:00+00:00
[2024-02-25T10:17:02.230+0000] {standard_task_runner.py:57} INFO - Started process 120 to run task
[2024-02-25T10:17:02.237+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'load_data_from_csv', 'csv_to_stg', 'scheduled__2024-02-25T10:16:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/dag_raw_to_dds.py', '--cfg-path', '/tmp/tmpr83__ecb']
[2024-02-25T10:17:02.240+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask csv_to_stg
[2024-02-25T10:17:02.314+0000] {task_command.py:410} INFO - Running <TaskInstance: load_data_from_csv.csv_to_stg scheduled__2024-02-25T10:16:00+00:00 [running]> on host 44b084e90875
[2024-02-25T10:17:02.411+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_data_from_csv' AIRFLOW_CTX_TASK_ID='csv_to_stg' AIRFLOW_CTX_EXECUTION_DATE='2024-02-25T10:16:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-25T10:16:00+00:00'
[2024-02-25T10:17:02.419+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_raw_to_dds.py", line 16, in csv_load
    df = pd.read_csv('data/fraud.csv')
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: 'data/fraud.csv'
[2024-02-25T10:17:02.434+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=load_data_from_csv, task_id=csv_to_stg, execution_date=20240225T101600, start_date=20240225T101702, end_date=20240225T101702
[2024-02-25T10:17:02.443+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 18 for task csv_to_stg ([Errno 2] No such file or directory: 'data/fraud.csv'; 120)
[2024-02-25T10:17:02.494+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2024-02-25T10:17:02.526+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
